### ISSUE TITLE
License of gt6fission.pdf

### ISSUE THREAD

**Issue by Tanimodori** [GregTech6/gregtech6#80]
Hi Greg, I'm working on remaking my previous nuclear fission simulator and planning to add an introduction for it. I find out that Erik3003's document (`https://mechaenetia.com/1.7.10/gt6fission.pdf`) helps a lot in explaining the machanism and want to adapt it into my guide. So under what license that particular document was released? Is it CC0 ^ ^?

**Comment by GregoriusT** [comment]
That is a very good Question, does it say anywhere in the Document itself?

Let me ping @Erik3003

**Comment by Tanimodori** [comment]
No, the document says nothing about its license. I've searched his original post, the forum, the changelog and over the Internet. The only related result I found is [this post](https://www.mcmod.cn/post/1217.html) by FourierTf where he translated `gt6fission.pdf` and specify a "CC-BY-NC-SA 3.0" license. But I assumed that CC license was chosen arbitrarily by the translator other than the original author Erik because there's no evidence to show that Erik has ever put a license on the pdf or permitted someone to translate it. So the CC license may be invalid if Erik ends up with an incompatible license (e.g. CC-ND or proprietary ones ^  ^).

Let's wait for Erik to answer then.
